### PR TITLE
Feature lwdev 6875 Update columns order in the "orders" and "trades" tabs

### PR DESCRIPTION
**Pull request by GeorgePuisha**
https://lykkex.atlassian.net/projects/LWDEV/issues/LWDEV-6875

### CODE REVIEW

File: src/components/OrderList/OrderListItem.tsx [hunk 1]
```diff
@@ -25,27 +27,42 @@ const OrderListItem: React.SFC<OrderActions & OrderListItemProps> = ({
...
+    displayName,
+    accuracy,
+    baseAsset: {accuracy: baseAssetAccuracy, name: baseAssetName},
+    quoteAsset: {accuracy: quoteAssetAccuracy, name: quoteAssetName}
+  }
 }) => {
   const handleEditOrder = () => onEdit(id);
   const handleCancelOrder = () => cancelOrder(id);
+
+  const value =
```
Comment (optimusway) [inline src/components/OrderList/OrderListItem.tsx]: i believe you should define `computed` prop at the model level

File: src/components/OrderList/Orders.tsx [hunk 2]
```diff
@@ -59,45 +59,42 @@ class Blotter extends React.Component<OrdersProps, TableSortState> {
...
+        className: 'right-align',
+        key: 'price',
+        value: 'Price'
+      },
       {
         sortDisabled: checkDataForSorting(this.state.data, 'volume'),
         className: 'right-align',
-        key: 'volume',
-        value: 'Volume'
+        key: 'amount',
```
Comment (optimusway) [inline src/components/OrderList/Orders.tsx]: does it work? it sorts by a _key_ prop and there is no amount prop in the `Order` or do i miss smth?

File: src/components/OrderList/index.tsx [hunk 3]
```diff
@@ -11,12 +11,9 @@ import OrderListItem from './OrderListItem';
 import Orders from './Orders';
 
 export const OrderCellWidth = {
-  Symbol: 70,
-  Id: 300,
-  Side: 50,
-  Filled: 100,
-  CreatedDate: 150,
-  Actions: 60
+  Symbol: 100,
```
Comment (optimusway) [inline src/components/OrderList/index.tsx]: is it your intention to remove "widths" for other columns?

File: src/components/Table/TitledCell.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+const withTitle = (Component: React.ComponentType<any>) => ({
+  children,
+  ...rest
+}: any) => (
+  <Component title={React.Children.toArray(children).join('')} {...rest}>
+    {children}
+  </Component>
+);
+
+export const TitledCell = withTitle(Cell);
```
Comment (optimusway) [inline src/components/Table/TitledCell.tsx]: good catch but this is done wrong

you need:
`export cont withTitle`
`export default withTitle(Cell)`
at least

then you don't omit `withTitle` which is also usable in many cases

File: src/components/OrderList/index.tsx (hunk 3 above)
Comment (GeorgePuisha) [reply src/components/OrderList/index.tsx]: Yes, I did it to make `Orders` and `Trades` tabs look similar.

File: src/components/Table/TitledCell.tsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+export const withTitle = (Component: React.ComponentType<any>) => ({
+  children,
+  ...rest
+}: any) => (
+  <Component title={React.Children.toArray(children).join('')} {...rest}>
+    {children}
+  </Component>
+);
+
+export default withTitle(Cell);
```
Comment (optimusway) [inline src/components/Table/TitledCell.tsx]: 👍

File: src/models/orderModel.ts [hunk 6]
```diff
@@ -21,6 +23,13 @@ class OrderModel {
     return this.filled !== 0 ? this.remainingVolume / this.volume : 0;
   }
 
+  @computed
```
Comment (optimusway) [inline src/models/orderModel.ts]: how can i test it? ;-)

Comment (optimusway) [reply src/models/orderModel.ts]: then you don't need to add `accuracy` in the order modek, which is also wrong coz accuracy means nothing to the `Order`. The order only knows about the instrument

File: src/components/OrderList/OrderList.tsx [hunk 7]
```diff
@@ -22,6 +22,7 @@ const OrderList: React.SFC<OrderListProps> = ({
       <tbody>
         {orders.map(order => {
           const instrument = getInstrumentById(order.symbol);
+          order.accuracy = instrument.quoteAsset.accuracy;
```
Comment (optimusway) [inline src/components/OrderList/OrderList.tsx]: this is a very bad practice to mutate state during render. i even don't know in all details how mobx deals with that, but the thing is that this is completely unneeded, see my comments about `value` and formatting

**Pull request merged**